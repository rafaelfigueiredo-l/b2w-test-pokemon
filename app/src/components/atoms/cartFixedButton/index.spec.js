import { BrowserRouter } from "react-router-dom";
import CartFixedButton from "./";

describe("<CartFixedButton />", () => {
  const component = (
    <BrowserRouter>
      <CartFixedButton link="http://www.something.com" />
    </BrowserRouter>
  );

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartFixedButton component", () => {
    expect(mounted.find(".cartFixedButton")).toHaveLength(2);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartFixedButton component", () => {
    expect(mounted.unmount()).toEqual({});
  });
});
