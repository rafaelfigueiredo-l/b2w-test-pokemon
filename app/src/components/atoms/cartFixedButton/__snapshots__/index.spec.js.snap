// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartFixedButton /> should mount correctly CartFixedButton component 1`] = `
<styled.p
  className="cartFixedButton"
>
  <p
    className="sc-AxjAm cpnBmN cartFixedButton"
  >
    <Link
      onClick={[Function]}
      to="http://www.something.com"
    >
      <LinkAnchor
        href="/http://www.something.com"
        navigate={[Function]}
        onClick={[Function]}
      >
        <a
          href="/http://www.something.com"
          onClick={[Function]}
        >
          Ver Carrinho
        </a>
      </LinkAnchor>
    </Link>
  </p>
</styled.p>
`;
